woocommerce/wc-admin
# Update product details report to use getItems

To check product variable status in the product detail report, I introduced a `getProducts` (to match the pattern of `getOrders`, etc). #1438 introduced a `getItems` method instead.

This PR updates the product report to use it.

Branched off of #1481 since that PR contains another fix for the report.

### Detailed test instructions:

* Open the single product view of a normal product. Make sure it displays correctly.
* Open a single product view of a variation and make sure the variation filters display.
